pinot-common/src/main/java/org/apache/pinot/common/utils/RegexpPatternConverterUtils.java

```diff
@@ -64,27 +70,54 @@ public static String likeToRegexpLike(String likePattern) {
         break;
     }
 
-    String escaped = escapeMetaCharacters(likePattern.substring(start, end));
-    StringBuilder sb = new StringBuilder(escaped.length() + 2);
+    likePattern = likePattern.substring(start, end);
+    StringBuilder sb = new StringBuilder();
     sb.append(prefix);
-    sb.append(escaped);
-    sb.append(suffix);
 
+    // handling SQL wildcards by replacing them with corresponding regex equivalents
```

Comment:
Can you be more specific on what characters are wildchards in SQL? Please give a complete list.

Reply:
Ack, fixed it.